client/landing/stepper/declarative-flow/internals/steps-repository/newsletter-setup/index.tsx

```diff
@@ -6,13 +6,13 @@ import { createInterpolateElement } from '@wordpress/element';
 import { Icon } from '@wordpress/icons';
 import { useI18n } from '@wordpress/react-i18n';
 import React, { FormEvent, useEffect } from 'react';
+import { base64ImageToBlob } from 'calypso/../packages/onboarding/src/setup-tailored-site-after-creation';
```

Comment:
This should be 
```suggestion
import { base64ImageToBlob } from '@automattic/onboarding';
```

And if it's not exported in the package, it should be.

Reply:
The next commit will address it =) thank you!
🎖️